azurelinuxagent/ga/monitor.py

```diff
@@ -142,6 +144,17 @@ def init_sysinfo(self):
         except ProtocolError as e:
             logger.warn("Failed to get system info: {0}", e)
 
+        try:
+            vminfo = self.imds_client.get_compute()
+                self.sysinfo.append(TelemetryEventParam('SubscriptionId',
```

Comment:
This line appears to be indented incorrectly.

Reply:
It is, fixed.